core/templates/dev/head/domain/skill/SkillUpdateServiceSpec.ts

```diff
@@ -16,28 +16,46 @@
  * @fileoverview Unit tests for SkillUpdateService.
  */
 
+// TODO(YashJipkate) Remove the following block of unnnecessary imports once
+// SkillUpdateService.ts is upgraded to Angular 8.
+import { AudioLanguageObjectFactory } from
+  'domain/utilities/AudioLanguageObjectFactory.ts';
+import { AutogeneratedAudioLanguageObjectFactory } from
+  'domain/utilities/AutogeneratedAudioLanguageObjectFactory.ts';
+import { MisconceptionObjectFactory } from
+  'domain/skill/MisconceptionObjectFactory.ts';
+// ^^^ This block is to be removed.
+
 require('App.ts');
 require('domain/editor/undo_redo/UndoRedoService.ts');
 require('domain/exploration/SubtitledHtmlObjectFactory.ts');
-require('domain/skill/MisconceptionObjectFactory.ts');
 require('domain/skill/SkillObjectFactory.ts');
 require('domain/skill/SkillUpdateService.ts');
 
 describe('Skill update service', function() {
   var SkillUpdateService,
     SubtitledHtmlObjectFactory,
     SkillObjectFactory,
-    MisconceptionObjectFactory,
+    misconceptionObjectFactory,
```

Comment:
initialize this with null. (for others as well.)

Reply:
Done!